1. Basic Web Development/frontend/html/index.html

```diff
@@ -35,7 +75,7 @@
         <div class="editor no-display">
             <div id="creation-page" class="">
                 <a class="to-edit no-decor" href="#edit">To edit ></a>
-                <h1>Design your preferences pop-up</h1>
+                <h1>Add new permissions</h1>
                 <form id="creation-form">
                     <h2>Category</h2>
                     <div>
```

Comment:
This is still not solved: https://github.com/dochiasebastian/workshops/pull/8#discussion_r670421687

Reply:
I still don't know what alternative to `div` I can use that is more specific and is also a block. I will further investigate it and will update when I find a solution.